src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyTextFilter.java

```diff
@@ -229,7 +228,7 @@ protected void finishLocalSetup() {
      * @throws IllegalStateException if the file could not be read.
      */
     private static FileText getFileText(String fileName) {
-        final Path path = Paths.get(fileName);
+        final Path path = Path.of(fileName);
```

Comment:
If you want to at least something to be merged, please move such changes to separate PR to just improve coding with rewrite dependency